### DISCUSSION TITLE
Property type not supported in Livewire for property: [{}]

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#13911] [Help]
### Package

Panel builder

### Package Version

v3.2.97

### How can we help you?

After creating a new resource, when I go to the edit page or the view page, this problem occurs. There is no problem on the Create page.

**Comment by leandrocfe** [comment]
Does it happen in a Filament fresh install?

**Reply by ArtMin96** [reply]
I have installed a new Laravel and I am trying to create a plugin.

```php
class Monitor implements Plugin
{
    public function getId(): string
    {
        return 'monitor';
    }

    public function register(Panel $panel): void
    {
        $panel
           ->resources([
                MonitorResource::class,
            ]);
    }

    public function boot(Panel $panel): void
    {
        // TODO: Implement boot() method.
    }
}
```

And the resource is a standard filament resource.

**Reply by ArtMin96** [reply]
I have installed fresh Laravel and Filament. I have the same problem, even though I don't create it as a plugin.

**Reply by ArtMin96** [reply]
It turns out the problem comes from Spatie's Monitor model. It has a `getUrlAttribute` method which probably conflicts with the Filament resource.

**Reply by jappi00** [reply]
@ArtMin96 I have the same problem. Any Idea how to fix that?
